website/src/pages/en/subgraphs/explorer.mdx

```diff
@@ -161,69 +181,77 @@ In the Epochs section, you can analyze on a per-epoch basis, metrics such as:
+- Click your wallet address in the top right corner
+- Your wallet acts as your user profile
+- In your profile dashboard, you can view and interact with several useful tabs
 
-### Profile Overview
+### Step 2. Explore the Tabs
+
+#### Profile Overview
 
 In this section, you can view the following:
 
-- Any of your current actions you've done.
-- Your profile information, description, and website (if you added one).
+- Your activity
+- Your profile information: total query fees, total shares value, owned stake. stake delegating
 
 ![Explorer Image 10](/img/Profile-Overview.png)
 
-### Subgraphs Tab
+#### Subgraphs Tab
 
-In the Subgraphs tab, you’ll see your published Subgraphs.
+The Subgraphs tab displays all your published Subgraphs.
 
-> This will not include any Subgraphs deployed with the CLI for testing purposes. Subgraphs will only show up when they are published to the decentralized network.
+> Subgraphs deployed with the CLI for testing purposes will not show up here. Subgraphs will only show up when they are published to the decentralized network.
 
 ![Explorer Image 11](/img/Subgraphs-Overview.png)
 
-### Indexing Tab
+#### Indexing Tab
 
-In the Indexing tab, you’ll find a table with all the active and historical allocations towards Subgraphs. You will also find charts where you can see and analyze your past performance as an Indexer.
+> If you haven't indexed, you see links to stake to index Subgraphs and browse Subgraphs on Graph Explorer.
 
-This section will also include details about your net Indexer rewards and net query fees. You’ll see the following metrics:
+The Indexing tab displays a table where you can review active and historical allocations to Subgraphs.
 
-- Delegated Stake - the stake from Delegators that can be allocated by you but cannot be slashed
-- Total Query Fees - the total fees that users have paid for queries served by you over time
-- Indexer Rewards - the total amount of Indexer rewards you have received, in GRT
-- Fee Cut - the % of query fee rebates that you will keep when you split with Delegators
-- Rewards Cut - the % of Indexer rewards that you will keep when splitting with Delegators
-- Owned - your deposited stake, which could be slashed for malicious or incorrect behavior
+Track your Indexer performance with visual charts and key metrics, including:
 
-![Explorer Image 12](/img/Indexer-Stats.png)
+- Delegated Stake - Stake from Delegators that can be allocated by you but cannot be slashed
+- Total Query Fees - Cumulative fees from served queries
+- Indexer Rewards (in GRT) - Total rewards earned
+- Fee Cut & Rewards Cut - the % of query fee rebates and Indexer rewards you'll keep when you split with Delegators
+- Owned Stake - your deposited stake, which could be slashed for malicious or incorrect behavior
```

Comment:
Should these hyphens also be replaced with `:` like you did above (under `#### Indexers` > `Specifics`)?

Reply:
Also, should these list items end with periods?